redis/redis.go

```diff
@@ -55,6 +55,11 @@ func (c *Component) Start() error {
 
 	c.Connection = redis.NewClient(opts)
 
+	_, err = c.Connection.Ping(context.Background()).Result()
+	if err != nil {
+		return err
+	}
+
 	return nil
```

Comment:
How about simplifying this a bit?

```suggestion
	return c.Health()
```